website/content/en/docs/deployment/add-ons/storage/efs/guide.md

```diff
@@ -239,6 +239,11 @@ yq e '.spec.template.spec.volumes[0].persistentVolumeClaim.claimName = env(CLAIM
 kubectl apply -f $GITHUB_STORAGE_DIR/notebook-sample/set-permission-job.yaml
 ```
 
+#### Automated setup default EFS owner  
```

Comment:
This is not necessarily limited to the automated setup since we could potentially run the manual dynamic provisioning steps with these options too, so could you move this documentation to a more generic spot.

Reply:
Yes I agree, see https://github.com/awslabs/kubeflow-manifests/pull/268#issuecomment-1169450072

Reply:
Done @mbaijal